homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+from typing import Any
+import logging
+
+from homeassistant.components.fan import FanEntity, FanEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the Lutron fans."""
+    devs = []
+    for area_name, device in hass.data[LUTRON_DEVICES]["fan"]:
+        dev = LutronFan(
+            area_name, 
+            device, 
+            hass.data[LUTRON_CONTROLLER],
+            device.uuid, 
+            device.name, 
+            None)
+        devs.append(dev)
+
+    add_entities(devs, True)
```

Comment:
```suggestion
    entities = [
        LutronFan(
            area_name, 
            device, 
            hass.data[LUTRON_CONTROLLER],
            device.uuid, 
            device.name, 
            None)
        for area_name, device in hass.data[LUTRON_DEVICES]["fan"]
    ]
    add_entities(entities, True)
```

Reply:
this was written this way for consistency with the other entities.  I don't have a problem changing it, but it would see better to keep the code consistent in the integration.  Thoughts?

Reply:
It should be written better here and fix the others then in a follow-up PR.
Probably when you switch to config flow which you already mentioned as a next step